README.md

```diff
@@ -1,5 +1,196 @@
 # terraform-aws-rds-aurora
-Terraform module which creates RDS Aurora resources on AWS
 
+Creates a AWS Aurora RDS cluster, instances, DB subnet group and optionally:
```

Comment:
> As a separate idea, how hard would it be to extend https://github.com/terraform-aws-modules/terraform-aws-rds module to support Aurora specific features?

I considered this but the problem is that although Aurora falls under the RDS umbrella, the resources are actually quite different. It's not just a different engine for the existing RDS resource. It has a cluster and then instances that are members of the cluster. It can have a large amount of cluster instances/replicas to scale read load, up to 15. It can autoscale. It has a serverless option. I think that trying to combine it into that existing module not ideal, the only resources it would share is the option group and subnet group. Then there would need to be 3 extra resources that are solely for aurora.

> This resource module should not care about anything expect required RDS specific resources, so there is no need to bring SNS, Cloudwatch, Route53, App Autoscaling, etc.

OK I can remove SNS/cloudwatch if you don't want to include basic monitoring. I can also remove route53 record.

Autoscaling should stay. This isn't "App Autoscaling", this is [scaling of the replicas](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Aurora.Integrating.AutoScaling.html) and is one of the coolest features of Aurora.

Reply:
Thanks for the detailed answer.

Yes, let's keep Aurora separately from the existing RDS module and include:
1. Monitoring (enhanced monitoring option in RDS, if there is such in Aurora)
2. Autoscaling